src/snagflash/dfu.py

```diff
@@ -32,22 +32,24 @@ def dfu_cli(args):
 	pid = int(dev_addr[1], 16)
 	dev = get_usb(vid, pid)
 	dev.default_timeout = int(args.timeout)
-	for dfu_config in args.dfu_config:
-		(altsetting,sep,path) = dfu_config.partition(":")
-		altsetting = int(altsetting)
-		with open(path, "rb") as file:
-			blob = file.read(-1)
-		size = len(blob)
-		print(f"Downloading {path} to altsetting {altsetting}...")
-		logger.debug(f"DFU config altsetting:{altsetting} size:0x{size:x} path:{path}")
+	altsetting = 0
+	if args.dfu_config:
+		for dfu_config in args.dfu_config:
```

Comment:
Could you please move the body of this `for` loop into a separate function? The indentation is getting a little too deep here.

Reply:
Fully refactored and reused for `--dfu-reset`